CHANGELOG.md

```diff
@@ -105,7 +105,40 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid Prime
 
-- Changed: The map tracker uses the same names for elevators as when editing a preset.
+- Fixed: Crashes in several rooms when pressing start to skip cutscene exactly 1 second from the end of cutscene
+- Fixed: Incorrect shield texture for vertical **Power Beam Only** doors
+- Fixed: Elite Research - Stutter when loading room
+- Fixed: Ruined Shrine Access - Still playing Beetle fight music after leaving
+- Fixed: Mine Security Station - Wave Pirates not dropping down if the player didn't skip the cutscene immediately
+- Fixed: Uncrashed Frigate - Escape music sometimes playing at inappropriate times
```

Comment:
Line 113
Not playing at inappropriate times, just stopping then playing again when left Reactor Core after PQ death